packages/audioplayers_windows/windows/Logger.h

```diff
@@ -0,0 +1,21 @@
+#include <string>
+
+#include <iostream>
+
+enum class LogLevel
+{
```

Comment:
unless there is some enum/cpp specific convention I don't know, I think this should follow the rule used by other classes (like Logger) or blocks right?
```suggestion
enum class LogLevel {
```

Reply:
actually I am seeing a mix of stuff here. let's merge as is and I will study c++ best practices/add an auto linter later